packages/cuba-react-core/src/app/MainStore.ts

```diff
@@ -108,7 +152,9 @@ export class MainStore {
     return !this.authenticated && !this.usingAnonymously;
   }
 
-  /**
+  // TODO same method as `isEntityDataLoaded`, either one shall be removed.
+  // TODO Perhaps `isEntityDataLoaded` is a better name as this method can not only be used  in the entity management screens.
+  /*
    * Define id mainStore is ready to serve entiry management components.
```

Comment:
Agree. This should be removed on merge paging. But I miss it. Could be replaced with `isEntityDataLoaded`.